pkg/controller/login/select_password.go

```diff
@@ -51,18 +59,72 @@ func (c *Controller) HandleSubmitNewPassword() http.Handler {
 			return
 		}
 
+		m := controller.TemplateMapFromContext(ctx)
+		f := flash.New(nil)
+
+		if err := c.validateComplexity(form.Password); err != nil {
+			logger.Errorw("validateComplexity failed", "error", err)
```

Comment:
Anything that's error or errorw will eventually show up in an alert. We should only be using error when it's a _server_ error (i.e. something we should be paged for).

Since we're returning this error to the user in the flash, it seems like we shouldn't log it at all.